### PR TITLE
moved schema folder out of auth in modules

### PR DESCRIPTION
**Pull request by atharva29**

### CODE REVIEW

**Comment by YourTechBud** [comment]
@atharva29 why did you delete the gitignore file?

**Comment by atharva29** [comment]
By mistake,  i will add that

File: main.go [hunk 1]
```diff
@@ -22,9 +22,10 @@ var essentialFlags = []cli.Flag{
 		EnvVar: "NODE_ID",
 	},
 	cli.StringFlag{
-		Name:  "config",
-		Value: "config.yaml",
-		Usage: "Load space cloud config from `FILE`",
+		Name:   "config",
+		Value:  "config.yaml",
+		Usage:  "Load space cloud config from `FILE`",
+		EnvVar: "CONFIG",
```
Comment (YourTechBud) [inline main.go]:
```suggestion
		EnvVar: "SC_CONFIG",
```

**Comment by atharva29** [comment]
I will correct it

**Pull request merged**